app/src/main/java/org/oppia/android/app/topic/TopicFragmentPresenter.kt

```diff
@@ -61,12 +64,20 @@ class TopicFragmentPresenter @Inject constructor(
     this.internalProfileId = internalProfileId
     this.topicId = topicId
 
-    binding.topicToolbar.setNavigationOnClickListener {
+    binding.topicToolbar.getChildAt(0).setOnClickListener {
```

Comment:
Could you tell me why you made this change?

Reply:
Since there is no individual view for toolbar title, setting 'onNavgation..listener' will make the whole toolbar clickable which talkback will read "double click.."- While we want to keep the navigation function we can use getChildAt(0) to set the listener to just nav icon.

Also added this to explanation

Reply:
Mhh, and does the marquee effect still work when talkback is off?

Reply:
It is a one-time action in the handleOnCreate function.

My thought on this is that, if someone needs to use accessibility manager, they will turn it on before even entering the app. I cannot figure out a reason for users to turn on the accessibility manager in half of the way like how we test the app.

My thought on this is if we want to switch between the talkback and the marquee effect within the activity, we may need to enable the listener in onResume and disable it in onStop. But I am not sure how to control this in a presenter.

Reply:
My question is actually whether the marquee functionality works for user who are not using the talkback.

Reply:
Yes, it works